app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    try:
+        _, source, destination = command.split()
+    except ValueError:
+        print("Error!")
+
+    if source == destination:
+        return
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination += source
```

Comment:
When concatenating paths, it's recommended to use `os.path.join` to ensure cross-platform compatibility.